### ISSUE TITLE
Add rule enforcing operator imports from "rxjs" instead of "rxjs/operators"

### ISSUE THREAD

**Issue by JasonWeinzierl** [JasonWeinzierl/eslint-plugin-rxjs-x#9]
Since RxJS v7.2.0, operators should be imported from "rxjs" instead of "rxjs/operators": https://rxjs.dev/guide/importing.  We should add a rule for enforcing this.

Related: #7 , https://github.com/ReactiveX/rxjs/pull/6488, https://github.com/cartant/eslint-plugin-rxjs/issues/108, https://github.com/cartant/eslint-plugin-rxjs/pull/121

**Comment by rob4226** [comment]
I've been using the below rule I configured myself but it would definitely be nice to have a rule included with this plugin.

```js
'@typescript-eslint/no-restricted-imports': [
  'error',
  {
    paths: [
      {
        name: 'rxjs/Rx',
        message: 'rxjs/Rx is no longer a valid import site. This is very old import path for RxJS, use `from rxjs`.',
      },
      {
        name: 'rxjs/operators',
        message: "With RxJS v7.2.0, most operators have been moved to 'rxjs' export site. This means that the preferred way to import operators is from 'rxjs', while 'rxjs/operators' export site has been deprecated. https://rxjs.dev/guide/importing#new-in-rxjs-v720",
      },
],
```
